Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.yml

```
@@ -1815,6 +1911,12 @@ script:
       required: true
     description: View the file retrieved by the core-retrieve-files command according to the action ID. Before running this command, you can use the core-action-status-get command to check if this action completed successfully.
     name: core-retrieve-file-details
+    supportedModules:
+      - C3
+      - X0
+      - X3
+      - X5
+      - ENT_PLUS
     outputs:
     - contextPath: File
       description: The file details command results.
```

```suggestion
      description: The file details returned by the command.
```
The description isn't too clear to me, so it was hard to edit. Could you check if the sentiment is the same as the original?